internal/policy/numerical.go

```diff
@@ -66,7 +66,7 @@ func (p *Numerical) Latest(versions []string) (string, error) {
 
 		switch {
 		case i == 0:
-			break // First iteration, nothing to compare
+			// First iteration, nothing to compare
```

Comment:
I like the explicit `break`, since it disambiguates for people who have trouble remembering whether Go's `case` has fall-through or not (like me).

Reply:
I just went with the linter recommendation. I can definitely revert back to using break if you think it's worth it, no problem.

Reply:
I often find myself disagreeing with linters. Maybe I should consider .... rewriting all the linters ...

Reply:
Not a blocker, I leave it to your judgement @relu :-)